src/api/app/views/webui2/webui/project/_pulse_list_comments.html.haml

```diff
@@ -0,0 +1,10 @@
+- if @comments.any?
+  - people = @comments.group(:user_name).count
```

Comment:
this count is made more than once....

Reply:
Please elaborate what you mean. I don't understand.